components/Navbar.js

```diff
@@ -1,109 +1,90 @@
+      });
+    };
+  }, [menuActive]);
 
-	render() {
-		return (
-			<div id="navbar" role="banner">
-				<div id="navbar-inner">
-					<Link href="/">
-						<a id="nav-title" className="nav-section left">
-							{/* TODO: resolve next/image issue */}
-							{/* eslint-disable-next-line @next/next/no-img-element */}
-							<img src={'/images/acm_wordmark&logo.svg'} id="acm-logo" alt="ACM at UCLA Logo"></img>
-							{/* TODO: use next image without breaking deploy */}
-						</a>
-					</Link>
-					<div className="nav-section right" id="desktop-nav" role="navigation">
-						<ul className="nav-items">
-							<li><Link href="/about"><a>About</a></Link></li>
-							<li><Link href="/committees"><a>Committees</a></Link></li>
-							{/* <li><Link href="/gm"><a>{gmData.date.quarter} GM</a></Link></li> */}
-							<li><Link href="/events"><a>Events</a></Link></li>
-							<li><Link href="/sponsors"><a>Sponsors</a></Link></li>
-							<li><Link href="https://members.uclaacm.com"><a className="button button-transparent button-lg font-header">Member Login</a></Link>
-							</li>
-						</ul>
-					</div>
-					<div className="nav-section right" id="mobile-nav">
-						{/* TODO: resolve this by refactoring the navbar */}
-						{/* eslint-disable-next-line jsx-a11y/label-has-for */}
-						<label htmlFor="menu-toggle">
-							<div className="hamburger-icon">
-								<div className="bar" id="top-bar" />
-								<div className="bar" id="middle-bar" />
-								<div className="bar" id="bottom-bar" />
-							</div>
-						</label>
-						<input type="checkbox" id="menu-toggle" />
-						<div id="hamburger-menu" role="navigation">
-							<ul className="nav-items">
-								<li><Link href="/about"><a>About</a></Link></li>
-								<li><Link href="/committees"><a>Committees</a></Link></li>
-								{/* <li><Link href="/gm"><a>{gmData.date.quarter} GM</a></Link></li> */}
-								<MobileNavItem/>
-								<li><Link href="/events"><a>Events</a></Link></li>
-								<li><Link href="/sponsors"><a>Sponsors</a></Link></li>
-								<li><Link href="https://members.uclaacm.com"><a className="button button-transparent button-lg font-header">Member Login</a></Link>
-								</li>
-							</ul>
-						</div>
-					</div>
-				</div>
-			</div>
-		);
-	}
+  return(
+    <nav id="navbar">
+      <section id="nav-container">
+        <Link href="/">
+          <a id="nav-title" className="nav-section left" aria-label="acm home">
+            {/* TODO: resolve next/image issue */}
+            {/* eslint-disable-next-line @next/next/no-img-element */}
+            <img src={'/images/acm_wordmark&logo.svg'} id="acm-logo" alt="ACM at UCLA Logo"></img>
+            {/* TODO: use next image without breaking deploy */}
+          </a>
+        </Link>
+        <section id="nav-items-container">
+          <div className={menuActive} id="hamburger" role="button" aria-label="navigation menu" aria-expanded={menuActive === 'active' ? true : false} tabIndex="0">
```

Comment:
quick question - is there a reason that this is a `<div>` instead of a `<button>`? In general, it'd be great if we could use `<button>` instead, though I see you've set the `role` which helps with accessibility!